benchmark/pom.xml

```diff
@@ -24,28 +24,40 @@
     <version>9</version>
   </parent>
 
-  <!-- TODO: change group id when 9.0 is released -->
-  <groupId>com.netflix.feign</groupId>
```

Comment:
we still should be under the io.github.openfeign group ID, not the sonatype one. you'll want to re-add this I think